en/scale-a-tidb-cluster.md

```diff
@@ -16,9 +16,61 @@ Currently, the TiDB cluster supports management by Helm or by TidbCluster Custom
 
 ### Horizontal scaling operations (CR)
 
+#### Scale PD, TiDB, and TiKV
+
 Modify `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` in the `TidbCluster` object of the cluster to a desired value using kubectl.
 
-If TiFlash is deployed in the cluster, you can scale in and out TiFlash by modifying `spec.tiflash.replicas`.
+#### Scale out TiFlash
+
+If TiFlash is deployed in the cluster, you can scale out TiFlash by modifying `spec.tiflash.replicas`.
+
+#### Scale in TiFlash
+
+1. Expose the PD service by using `port-forward`:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl port-forward -n ${namespace} svc/${cluster_name}-pd 2379:2379
+    ```
+
+2. Open a **new** terminal tab or window. Check the maximum number (`N`) of replicas of all data tables that have enabled TiFlash by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl 127.0.0.1:2379/pd/api/v1/config/rules/group/tiflash | grep count
+    ```
+
+    In the printed result, the largest value of `count` is the maximum number (`N`) of replicas of all data tables.
+
+3. Go back to the terminal window in Step 1, where `port-forward` is running. Press <kbd>Ctrl</kbd>+<kbd>C</kbd> to stop `port-forward`.
+
+4. If after the scaling-in operation, the number of remaining Pods in TiFlash >= `N`, skip to Step 6. Otherwise, take the following steps:
```

Comment:
```suggestion
4. After the scale-in operation, if the number of remaining Pods in TiFlash >= `N`, skip to Step 6. Otherwise, take the following steps:
```